tools/i686-w64-mingw32.cmake

```diff
@@ -0,0 +1,14 @@
+set(TOOLCHAIN_PREFIX i686-w64-mingw32)
```

Comment:
We can reduce duplication by making one of the toolchains a symlink to another and then guess `TOOLCHAIN_PREFIX` based on `CMAKE_CURRENT_LIST_FILE`.  That was my first solution, but I recalled that without Developer Mode enabled Windows users could encounter problems with the symlinks.

Reply:
This and the other "toolchain" file looks, to me, like system- and build-specific config, with no business being shipped; I think move these to `.github` (`.github/workflows`?), and maybe merge them (looks like they're identical except for `TOOLCHAIN_PREFIX` and `CMAKE_SYSTEM_PROCESSOR`, which maybe can go into `-D`?) or inline them as `-D` wholesale.